gef.py

```diff
@@ -664,6 +664,21 @@ def from_info_mem(cls, perm_str: str) -> "Permission":
         if "x" in perm_str: perm |= Permission.EXECUTE
         return perm
 
+    @classmethod
+    def from_filter_repr(cls, filter_str: str) -> list["Permission"]:
+        perms = [cls(0)]
```

Comment:
Type hints need to be expicit here

```suggestion
        perms: list["Permission"] = [cls(0)]
```